src/vivarium/framework/results/observation.py

```diff
@@ -2,9 +2,20 @@
 ============
 Observations
 ============
-"""
 
-from __future__ import annotations
+An observation is a class object that records simulation results; they are responsible
+for initializing, gathering, updating, and formatting results.
+
+The provided :class:`BaseObservation` class is an abstract base class that should
+be subclassed by concrete observations. While there are no required abstract methods
+to define when subclassing, the class does provide common attributes as well
+as an `observe` method that determines whether to observe results for a given event.
```

Comment:
Shouldn't this say a `to_observe` method?

Reply:
Not quite. There _is_ a `to_observe` attribute (which is a Callable that returns a boolean), but I'm highlighting the `observe` method that is really the thing that calls a given observation's `results_gatherer` method (if `to_observe()` returns true for a given event)